File: helfomat-api/src/test/java/de/helfenkannjeder/helfomat/api/TestApplication.java
package de.helfenkannjeder.helfomat.api;

import de.helfenkannjeder.helfomat.core.organisation.OrganisationRepository;
import de.helfenkannjeder.helfomat.core.question.QuestionRepository;
import de.helfenkannjeder.helfomat.core.template.OrganizationTemplateRepository;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.test.mock.mockito.MockBean;

/**
 * @author Valentin Zickner
 */
@SpringBootApplication
public class TestApplication {

    @MockBean
    OrganisationRepository organisationRepository;

    @MockBean
    OrganizationTemplateRepository organizationTemplateRepository;

    @MockBean
    QuestionRepository questionRepository;

}

File: helfomat-api/src/test/kotlin/de/helfenkannjeder/helfomat/api/TestApplication.kt
package de.helfenkannjeder.helfomat.api

import de.helfenkannjeder.helfomat.core.organization.OrganizationRepository
import de.helfenkannjeder.helfomat.core.question.QuestionRepository
import de.helfenkannjeder.helfomat.core.template.OrganizationTemplateRepository
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.test.mock.mockito.MockBean

/**
 * @author Valentin Zickner
 */
@SpringBootApplication
open class TestApplication {
    @MockBean
    lateinit var organizationRepository: OrganizationRepository

    @MockBean
    lateinit var organizationTemplateRepository: OrganizationTemplateRepository

    @MockBean
    lateinit var questionRepository: QuestionRepository

    @MockBean
    lateinit var captchaValidator: CaptchaValidator

}